File: assignment19.py
str_list = ["lorem", "ipsum", "colbern", "three"]   

def stringLengthFinder(str_list):
    empty_list = []
    for i in str_list:
        empty_list.append(len(i))

    print(empty_list)

stringLengthFinder(str_list)

File: javascript_assignments/assignment19.js
str_list = ["lorem", "ipsum", "colbern", "three"]

function stringLengthFinder(str_list){
    let empty_list = []
    for (let i = 0; i < str_list.length; i++){
        empty_list.push(str_list[i].length)
    }
    console.log(empty_list)
}

stringLengthFinder(str_list)